README.md

```diff
@@ -157,7 +157,7 @@ report_to_stdout = true
 acceptance = "undefined"
 
 [automate]
-enable = false
+enable = true
```

Comment:
Please set this to false by default. Otherwise this will fill out someone's reporting information with failing data.

Reply:
Can you clarify this point? When I have it set to `false` it doesn't report to Automate at all.

Reply:
This also feels like we're mashing information about the scaffolding and the example together. For the example, you 100% need a `default.toml` in your hab package otherwise, InSpec doesn't know where to send the results.

Maybe we should consider breaking the readme up into separate files, one for the scaffolding and others for each example type. That might reduce the confusion between the different bits in this repo.

Reply:
You don't need a `default.toml` in your habitat package as it is built by the scaffolding with the defaults. It is my opinion that we should instead add a readme that tells a user how to properly override the default settings through a `user.toml` or `hab config apply`.

Reply:
You want it disabled by default, because whoever takes this example will need to fill out their internal automate server information before enabling reporting. Otherwise, if reporting is turned on and the default values are used, you will receive a runtime error.

We should always ship working default configurations that don't fail. It's up to the user to enable automate if they so choose.